src/vorta/store/settings.py

```diff
@@ -92,9 +92,13 @@ def get_misc_settings() -> List[Dict[str, str]]:
     else:
         settings += [
             {
-                'key': 'enable_background_question', 'value': True, 'type': 'checkbox',
+                'key': 'enable_background_question',
+                'value': True,
+                'type': 'checkbox',
                 'label': trans_late('settings',
-                                    'Display background exit dialog')
+                                    "If the system tray isn't available, "
+                                    "ask whether to continue in the background "
```

Comment:
Removed the dot at the end of the sentence and moved the space to the previous line. So we stick to your HIGs more?

Reply:
Right :smiley: